python/lsst/daf/butler/cli/utils.py

```diff
@@ -352,10 +352,10 @@ def split_kv(
     separator: str = "=",
     unseparated_okay: bool = False,
     return_type: type[dict] | type[tuple] = dict,
-    default_key: str = "",
+    default_key: str | None = "",
```

Comment:
This `None` is here specifically to support `--log-levels` because the code in `cliLog` is using `None` to indicate the default logger and "" or "." to indicate the root logger. They are not the same because we want `--log-level INFO` to affect the `lsst` logger and not the root logger.